dgraph/dgraph.go

```diff
@@ -17,21 +18,43 @@ type Dgraph struct {
 	Conn   *grpc.ClientConn
 }
 
-//New creates a new instance of DGraph
-func New(addr string) (*Dgraph, error) {
+//New creates a new instance of DGraphc
```

Comment:
what are username and password? Are those end users, ie, hypha members, or will this be called by a server side user / password?

Reply:
@n13 This is server side username and password. I've created a special user on DGraph Cloud for this service which has access just for methods it needs